src/Pim/Bundle/EnrichBundle/Normalizer/ProductViolationNormalizer.php

```diff
@@ -22,28 +20,21 @@ class ProductViolationNormalizer implements NormalizerInterface
      */
     public function normalize($violation, $format = null, array $context = [])
     {
-        $path = $violation->getPropertyPath();
+        $propertyPath = $violation->getPropertyPath();
 
-        if (0 === strpos($path, 'values')) {
+        if (0 === strpos($propertyPath, 'values')) {
```

Comment:
No, but I can look for `values[` to be more precise, as the propertyPath looks like that:
`values[{"code":"description","locale":"en_US","scope":"mobile"}].text` (.text for a text attrubute type).